manifest.json

```diff
@@ -1,7 +1,7 @@
 {
     "id": "obsidian-spaced-repetition",
     "name": "Spaced Repetition",
-    "version": "1.11.2",
+    "version": "1.11.1",
```

Comment:
Looks like there's some conflict here since we shouldn't we're undoing changes that have already gone in

Reply:
I am super sorry, that I caused this (I also read that this cause a mix up with my previous commit). It must have happened, while I was integrating the new commits from your master branch and as I didn't modify these files by hand, I maybe wasn't particularly looking out for mix ups in them 😕.

Reply:
All is back to what it was before.